AquariumShop.php

```diff
@@ -97,14 +130,20 @@ public function getSalePrice() : float {
 
 $aquarium1 = new Aquarium(100, 400);
 $aquarium1->addFish($angelfish, 10);
+$aquarium1->addGadget($gadgets[rand(0, sizeOf($gadgets)-1)]);
+var_dump($aquarium1);
 
 $aquarium2 = new Aquarium(60, 250);
 $aquarium2->addFish($jewelCichlid, 5);
 $aquarium2->addFish($kribensis, 10);
 $aquarium2->addFish($lionheadCichlid, 2);
+$aquarium2->addGadget($gadgets[rand(0, sizeOf($gadgets)-1)]);
```

Comment:
fyi sizeOf isn't commonly used. PHP world uses count(). sizeOf is just an alias. http://php.net/manual/en/function.count.php